### DISCUSSION TITLE
Having the item page refresh after a manual flow update has completed

### DISCUSSION THREAD

**Discussion by farmisen** [directus/directus#14585] [Feature Requests]
My (reduced) use case is the following:

I have setup a manually triggered flow to `Publish` an item of a given collection from the item page. The flow itself works perfectly and consist on a trigger block followed by an update block that change the `status` field content to "published" (but eventually will alter more fields).

![image](https://user-images.githubusercontent.com/229946/180577602-b98de299-5c44-4bdb-b307-c13885738843.png)

My issue is that when I trigger the flow from the item page, the item values do not update like they do after a ⌘+S and after the flow has completed, I have to manually reload the page to see the correctly updated value.

Is there any official way to achieve that? Any work around you can think of? This is a relatively critical issue for the thingie I am working on and I will gratefully welcome any suggestion pointing me toward the correct direction. Thanks!

**Comment by rijkvanzanten** [comment]
There's currently no configuration for that, but I agree it's important to have. We should also warn the user for any current active changes that they might lose when the flow is executed 👍🏻

**Reply by rijkvanzanten** [reply]
This is will be implemented and is tracked through https://github.com/directus/directus/issues/14630

**Comment by azrikahar** [comment]
#14630 has been resolved 👍
